cs/forms.texy

```diff
@@ -858,6 +873,37 @@ Když do vykreslování zapojíme ještě [Latte |default-macros#toc-formulare],
 {/form}
 \--
 
+Velmi snadno také můžete propojit formulář s existující šablonou. Stačí jen doplnit atributy n:name:
+
+/--php
+function createComponentSignInForm()
+{
+    $form = new Form;
+    $form->addText('user')->setRequired();
+    $form->addPassword('password')->setRequired();
+    $form->addSubmit('send');
+    return $form;
+}
+\--
+
+/--
+<form n:name=signInForm class=form>
+    <p><label n:name=user>Username: <input n:name=user size=20></label>
+    <p><label n:name=password>Password: <input n:name=password></label>
+    <p><input n:name=send class="btn btn-default">
+</form>
+\--
+
+Atribut `n:name` lze používat i s elementy `<select>`, `<button>` nebo `<textarea>`.
+
+Dále můžete vykreslovat prvky jako je RadioList, Checkbox nebo CheckList pěkně po jednotlivých HTML elementech. Říká se tomu partial rendering:
+
+{foreach $form[gender]->items as $key => $label}
+    <label n:name="gender:$key"><input n:name="gender:$key"> {$label}</label>
+{/foreach}
+
+Nebo lze použít klasická makra `{input gender:$key}` a `{label gender:$key}`, trik je tom názvu s dvojtečkou.
```

Comment:
jeste bych tu zminil syntaxi pro obycnejny checkbox, tedy `{input foo:}`, z tehle prikladu to nemusi byt patrne.